js/apporder.js

```diff
@@ -61,4 +75,29 @@ $(function () {
 			});
 		}
 	});
+
+	$(".apporderhidden").change(function(){
+		var hiddenList = [];
+		var url;
+		var type = $("#appsorter").data("type");
+
+		if(type === 'admin') {
+			url = OC.generateUrl('/apps/apporder/saveDefaultHidden');
+		} else {
+			url = OC.generateUrl('/apps/apporder/savePersonalHidden');
+		}
+
+		$(".apporderhidden").each(function(i, el){
+			if(!el.checked){
+				hiddenList.push($(el).siblings('a').attr('href'))
```

Comment:
href has been removed in #37 

See: https://github.com/juliushaertl/apporder/pull/44/files#diff-df3a9cfbbde34fd0ef6162d314290fcdR54